src/plugins/passthrough/passthrough_impl.cpp

```diff
@@ -0,0 +1,130 @@
+#include <functional>
+#include "passthrough_impl.h"
+#include "system.h"
+#include "global_include.h"
+
+namespace mavsdk {
+
+PassthroughImpl::PassthroughImpl(System& system) : PluginImplBase(system)
+{
+    _parent->register_plugin(this);
+}
+
+PassthroughImpl::~PassthroughImpl()
+{
+    _parent->unregister_plugin(this);
+}
+
+void PassthroughImpl::init() {}
+
+void PassthroughImpl::deinit() {}
+
+void PassthroughImpl::enable() {}
+
+void PassthroughImpl::disable() {}
+
+Passthrough::Result PassthroughImpl::send_message(const Passthrough::MavlinkMessage mavlink_message)
+{
+    mavlink_message_t message;
+
+    message.checksum = mavlink_message.checksum;
+    message.magic = mavlink_message.magic;
+    message.len = mavlink_message.len;
+    message.incompat_flags = mavlink_message.incompat_flags;
+    message.compat_flags = mavlink_message.compat_flags;
+    message.seq = mavlink_message.seq;
+    message.sysid = mavlink_message.sysid; 
+    message.compid = mavlink_message.compid;
+    message.msgid = mavlink_message.msgid;
+
+    for(int i = 0; i < 33; i++) {
```

Comment:
Why 33?

Reply:
33 is the fixed size of the array for payload.